Classes/Emogrifier.php

```diff
@@ -155,6 +155,17 @@ class Emogrifier
     private $shouldKeepInvisibleNodes = true;
 
     /**
+     * Determines whether media query style blocks should be added to the body versus the head of the HTML.
+     * 
+     * If set to true, the <style> blocks will be added to the <body>
+     * 
+     * If set to false, the <style> blocks will be added to the <head>
+     *
+     * @var bool
+     */
+    private $shouldAddStyleToBody = false;
```

Comment:
I think we can make this new behavior the default instead of making it configurable. @Gabe-SpotMe, @jjriv What do you think?